pkg/controller/metrics/provision_underway_collector.go

```diff
@@ -96,14 +105,132 @@ func (cc provisioningUnderwayCollector) Describe(ch chan<- *prometheus.Desc) {
 	prometheus.DescribeByCollect(cc, ch)
 }
 
+var (
+	metricClusterDeploymentProvisionUnderwaySecondsDesc = prometheus.NewDesc(
+		"hive_cluster_deployment_provision_underway_seconds",
+		"Length of time a cluster has been provisioning.",
+		[]string{"cluster_deployment", "namespace", "cluster_type", "condition", "reason", "platform", "image_set"},
+		nil,
+	)
+)
+
 func newProvisioningUnderwayCollector(client client.Client) prometheus.Collector {
```

Comment:
Do we need to keep this old function? Seems like we could just transition fully to the new.

Reply:
updated to use the previous function to setup the collect instead of a new one, instead of dropping in favor of new one